.github/actions/build-node/action.yml

```diff
@@ -1,50 +0,0 @@
-name: Agoric SDK build Node.js
-description: 'Build Node.js and cache built files'
-
-inputs:
-  node-version:
-    description: 'The version of Node.js to use'
-    required: true
-  cache-built:
-    description: 'Cache the result of the build'
-    type: boolean
-    required: false
-    default: true
-
-runs:
-  using: composite
-  steps:
-    - uses: actions/setup-node@v1
-      with:
-        node-version: ${{ inputs.node-version }}
-    - name: cache node modules
-      uses: actions/cache@v1
-      with:
-        path: ~/.cache/yarn
-        key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
-        restore-keys: |
-          ${{ runner.os }}-yarn-
-
-    # 'yarn install' must be done at the top level, to build all the
-    # cross-package symlinks
-    - name: yarn install
-      run: yarn install
-      shell: bash
-    # We do a diff instead of using --frozen-lockfile because
-    # - it doesn't detect deletions from yarn.lock (see https://github.com/Agoric/agoric-sdk/pull/4646#discussion_r814034842)
-    # - the diff provides diagnostic info of what changed
-    - name: fail if git checkout differs
-      run: git diff --exit-code
-      shell: bash
-    - name: check dependencies
-      run: yarn check-dependencies
-      shell: bash
-    # 'yarn build' loops over all workspaces
-    - name: yarn build
-      shell: bash
-      run: yarn build
-
-    - uses: ./.github/actions/save-node
```

Comment:
We can remove `save-node` altogether now, right?

Reply:
> I'm still a little confused as to the steps order. Won't `restore-node` be run by various workflows in parallel, negating the cache benefits? I though we'd need to switch to a triggered workflow mechanism to fully take advantage of this.

It's alright for running workflows in parallel... there may not be many benefits to the caching in this case, but it will speed up reruns.  The speed benefits of parallelism are still useful.  The sequential workflows (like 'test-all-packages') are the ones that benefit from the caching.

It's a tradeoff, but the point is to preserve correctness regardless of when the workflows use `restore-node` or `restore-golang`.